utils/api.electron.js

```diff
@@ -0,0 +1,19 @@
+import electron from 'electron'
+
+let electronStore = electron.remote.getGlobal('sharedObj').electronStore
+
+var checkRedirectToWelcomePage = function () {
```

Comment:
Naming of the file is not good. Let it be store or prefs, also the methods to and the key let it be show-welcome

Reply:
regarding the file name: my assumption here is that all APIs related to electron will be provided in this file. another set of files will be created for let's say android or ios. so it will be api.ios or api.android. and every api file exposes the same set of functions but implemented differently wrt each underlying tech (electron vs react native vs etc. )

regarding the key: I agree. changed it. will update the PR